linera-views/src/key_value_store_view.rs

```diff
@@ -112,7 +123,9 @@ where
                     batch.put_key_value_bytes(key, value);
                 }
             }
+            true
         } else {
+            let do_something = !self.deleted_prefixes.is_empty() || !self.updates.is_empty();
```

Comment:
Do we really need that variable, or could we just return here?

```suggestion
            if self.deleted_prefixes.is_empty() && self.updates.is_empty() {
                return Ok(());
            }
```

Or are there cases where the hash has to be recomputed even though nothing was written?

Reply:
(Also you'd need to be more specific than "something")

Reply:
Ok, I changed accordingly the code.